core/debug/src/core_armv7m/mpu_kinetis/debug_kinetis.c

```diff
@@ -95,48 +70,10 @@ static void debug_fault_mpu(void)
         }
     }
     else
-        dprintf("* No MPU violation found\n\r");
-    dprintf("\n\r");
-}
-
-void debug_map_addr_to_periph(uint32_t address)
-{
-    uint32_t aips_addr;
-    const MemMap *map;
-
-    dprintf("* MEMORY MAP\n");
-    dprintf("  Address:           0x%08X\n", address);
-
-    /* find system memory region */
-    if((map = memory_map_name(address)) != NULL)
     {
```

Comment:
You've added a `{` after the else. This is good, but should be done in a separate commit so that the changes are easier to read.